features/paint.yml

```diff
@@ -0,0 +1,9 @@
+name: paint()
+description: The `paint()` CSS function paints an element's background or border using a custom paint worklet.
+spec: https://drafts.css-houdini.org/css-paint-api-1/
+caniuse: css-paint-api
+compat_features:
+  - css.types.image.paint
+  - api.CSS.paintWorklet_static
+  - api.PaintWorkletGlobalScope
+group: css
```

Comment:
a real nit here (and I should write a script to format these automatically) but, if you're editing this file anyway, then moving this after `spec` will make this more consistent with other entries.